fileManager1API.js

```diff
@@ -82,7 +82,7 @@ var FileManager1Client = class DashToDock_FileManager1Client {
     getWindows(location) {
         let ret = new Set();
         for (let [k,v] of this._locationMap) {
-            if (k.startsWith(location)) {
+            if ((k + '/').startsWith(location + '/')) {
```

Comment:
Can we say for sure that neither k nor location will ever end with a '/'? This logic will break if that happens.

Reply:
Thanks for the review.

This would indeed break the trash indicator if you browse to a trashed directory, which results in a `k` value of: `trash:///folder1` and a `location` of: `trash:///`.

I've added a check to make sure that `location` doesn't end with a '/'.

`k` will end with a '/' if you browse to the file-system root (e.g `file:////`), the trash root (`trash:////`) or the root of some other location, but I cant't see how that would break the current logic.